tests/client_tester.py

```diff
@@ -0,0 +1,68 @@
+"""
+This is used for testing basic functionality of the test client.
+To run change the desired flags below and use the following command from the tap-tester repo:
+  'python ../tap-adroll/tests/client_tester.py'
+"""
+from test_client import TestClient
+
+
+##########################################################################
+# Testing the TestCLient
+##########################################################################
+if __name__ == "__main__":
```

Comment:
Why is this not a unit test case class? Why is it a script like this?

Reply:
This is just a basic script I threw together to aid in dev/debug of the test_client. It helps me do sanity checks and verify root cause for errors in posts and puts, without having to run an entire test